### PR TITLE
Refactor Search ITs into UTs

### PR DESCRIPTION
**Pull request by Kashif-S**

### CODE REVIEW

**Comment by djcooke** [comment]
Bad rebase?

**Comment by Kashif-S** [comment]
Fixed now.

File: sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateLibraryDilutionDao.java [hunk 1]
```diff
@@ -175,7 +175,9 @@ public String propertyForDate(Criteria item, DateType type) {
 
   @Override
   public String propertyForUserName(Criteria item, boolean creator) {
-    return creator ? "dilutionUserName" : "derivedInfo.lastModifier.loginName";
+    item.createAlias("creator", "creator");
+    item.createAlias("lastModifier", "lastModifier");
```
Comment (Kashif-S) [inline sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateLibraryDilutionDao.java]: I'm a little unsure if I put these `createAlias` calls in the right spot. The tests produce errors if this isn't here, but none of the other Daos, which use the same return statement as this function, use `createAlias` here. If there's a better spot to put these `createAlias` calls, please let me know.

Comment (djcooke) [reply sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateLibraryDilutionDao.java]: Add "creator" and "lastModifier" to the `STANDARD_ALIASES` list at the top of this class. These get added to all the criteria used in PaginatedDataSource

**Pull request merged**